.github/workflows/synronize_scs_docs.yml

```diff
@@ -0,0 +1,78 @@
+---
+name: Synchronize SCS Docs
+
+"on":
+  push:
+    paths:
+    # only trigger the workflow when docs.json has been changed
+      - 'docs.json'
+    branches:
+      - main
+
+jobs:
+  provide_repos_json:
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+    steps:
+      - name: Checkout the docusaurus repo (B)
+        uses: actions/checkout@v3
+
+      - name: Set Matrix
+        id: set-matrix
+        run: |
+          REPOS=$(echo $(cat ././docs.json) | sed 's/ //g' )
+          echo "::set-output name=matrix::$REPOS"
+  
+  sync_repos:
+    needs: provide_repos_json
+    runs-on: ubuntu-latest
+    strategy:
+      matrix: 
+        repos: ${{ fromJson(needs.provide_repos_json.outputs.matrix) }}
+    steps: 
+      - name: checkout the docusaurus repo (B)
+        uses: actions/checkout@v3
+
+      - name: set repos
```

Comment:
@maxwolfs this step can be removed as well

Reply:
Why can this be removed?

Reply:
true! is done

Reply:
erm, I meant not the "checkout the docusaurus repo (B)" step, but rather the "set repos" step ^^